BaseClasses.py

```diff
@@ -156,7 +163,7 @@ def __init__(self, players: int):
         self.start_inventory_from_pool: Dict[int, Options.StartInventoryPool] = {}
 
         for player in range(1, players + 1):
-            def set_player_attr(attr, val):
+            def set_player_attr(attr: str, val) -> None:
```

Comment:
still not great typing, but `val: Any`?

Reply:
`Any` means "We can't have better typing." or "It's not worth it to have better typing."

I think neither of those is the case for this. Better typing on this is possible, and it is worth it,
...just not right now, because this PR is for less invasive changes or more urgent changes.

If we put `Any` here right now, then I won't have the red squiggly to remind me to improve it later.